### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by KrisMur**
- [DEMO LINK](https://KrisMur.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://KrisMur.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Review by mira1zu — APPROVED** [review]

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,60 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+    <link rel="stylesheet" href="styles/index.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <img
+        class="product-card__image"
+        src="../src/images/imac.jpeg"
```
Comment (illiarerun) [inline src/index.html:17]: don't write redundant directories in path
```suggestion
        src="./images/imac.jpeg"
```

File: src/styles/stars.css [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--0 .stars__star:nth-child(n),
+.stars--1 .stars__star:nth-child(n+2),
+.stars--2 .stars__star:nth-child(n+3),
+.stars--3 .stars__star:nth-child(n+4),
+.stars--4 .stars__star:nth-child(n+5),
+.stars--5 .stars__star:nth-child(n+6) {
```
Comment (illiarerun) [inline src/styles/stars.css:24]: that's pretty much redundant here
```suggestion
{
```

File: src/styles/stars.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
+.stars {
+  display: flex;
+  height: 16px;
```
Comment (illiarerun) [inline src/styles/stars.css:3]: don't hardcode container height, let the content dictate it
```suggestion
```

File: src/styles/normalize.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  color: #060b35;
+}
+
+body {
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+}
```
Comment (illiarerun) [inline src/styles/normalize.css:17]: these are reset styles, move them to `reset.css`

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,60 @@
...
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product card</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
```
Comment (illiarerun) [inline src/index.html:10]: fix formatting here

**Review by illiarerun — CHANGES_REQUESTED** [review]
GJ, few little fixes left:

File: src/styles/reset.css [hunk 6]
```diff
@@ -0,0 +1,17 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  color: #060b35;
+}
```
Comment (jstmpelowycz) [inline src/styles/reset.css:9]: This is about normalizing

File: src/styles/reset.css [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+body {
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+}
```
Comment (jstmpelowycz) [inline src/styles/reset.css:17]: And this is about resetting

Comment (jstmpelowycz) [reply src/styles/reset.css:17]: Also, don't forget to combine styles if you can

**Review by jstmpelowycz — APPROVED** [review]
LGTM

**Pull request closed without merging**